docs/source/release/v5.1.0/direct_geometry.rst

```diff
@@ -33,4 +33,13 @@ Bugfixes
 - A bug in the validation of temporary workpaces for :ref:`MDNorm <algm-MDNorm>` has been fixed. One can now use temporary workspaces in any order when slicing.
 - Fixed bugs in the DGSPlanner interface, that cause errors when using negative values
 
+MSlice
+######
+
+Bugfixes
+--------
+
+- A bug causing an exception when attempting to overplot on a plot with different axes has been fixed.
```

Comment:
Could we reword it slightly to
"attempting to overplot workspaces with different axes has been fixed"